bin/eslint-github-init.js

```diff
@@ -68,14 +71,49 @@ inquirer.prompt(questions).then(answers => {
     eslintrc.extends.push('plugin:github/browser')
   }
 
-  if (answers.flow) eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'flow') eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'typescript') {
+    eslintrc.extends.push('plugin:github/typescript')
+
+    // TODO: Keep tsconfig.json and tslint.json as files in the lib and copy it over, rather then keeping it here inline.
+
+    const tsconfigPath = path.resolve(process.cwd(), 'tsconfig.json')
+    if (!fs.existsSync(tsconfigPath)) {
+      const tsconfigDefaults = {
+        compilerOptions: {
+          target: 'es5',
+          module: 'commonjs',
+          strict: true,
+          esModuleInterop: true
+        }
+      }
+      fs.writeFileSync(tsconfigPath, JSON.stringify(tsconfigDefaults, null, '  '), 'utf8')
```

Comment:
We don't need to create a `tsconfig.json` like we have to create `tslint.json` but I figured it would lead to less friction for developers.

Note that we don't create `.flowconfig` for flow.